lib/model/image_sscd.py

```diff
@@ -0,0 +1,70 @@
+from typing import Dict
+import io
+import urllib.request
+
+from lib.model.model import Model
+
+from pdqhashing.hasher.pdq_hasher import PDQHasher
+from lib import schemas
+from torchvision import transforms
+from PIL import Image
+import torch
+from lib.logger import logger
+import requests
+import numpy as np
+
+class Model(Model):
+    def compute_sscd(self, image_url: str) -> str:
+        """Compute perceptual hash using ImageHash library
+        :param im: Numpy.ndarray
+        :returns: Imagehash.ImageHash
+        """
+        # pdq_hasher = PDQHasher()
```

Comment:
should these pdq_hash things still be here?

Reply:
I've removed.